src/Engine/Audio/SteamAudioRenderers.cpp

```diff
@@ -0,0 +1,118 @@
+#include "SteamAudioRenderers.hpp"
+
+SteamAudioRenderers::SteamAudioRenderers() {
+    directEffect = nullptr;
+    envRenderer = nullptr;
+    binauralRenderer = nullptr;
+    binauralEffect = nullptr;
+}
+
+SteamAudioRenderers::SteamAudioRenderers(IPLhandle environment) {
+    this->environment = environment;
+    this->envRenderer = new IPLhandle;
+    this->directEffect = new IPLhandle;
+    this->binauralRenderer = new IPLhandle;
+    this->binauralEffect = new IPLhandle;
+    this->convEffect = new IPLhandle;
+
+    IPLRenderingSettings settings;
+    settings.frameSize = CHUNK_SIZE;
+    settings.samplingRate = SAMPLE_RATE;
+    settings.convolutionType = IPL_CONVOLUTIONTYPE_PHONON;
+
+    inputFormat.channelLayoutType = IPL_CHANNELLAYOUTTYPE_SPEAKERS;
+    inputFormat.channelLayout = IPL_CHANNELLAYOUT_MONO; //Only take mono input
+    inputFormat.numSpeakers = 1;
+    inputFormat.speakerDirections = NULL;
+    inputFormat.channelOrder = IPL_CHANNELORDER_INTERLEAVED;
+
+    outputFormat.channelLayoutType = IPL_CHANNELLAYOUTTYPE_SPEAKERS;
+    outputFormat.channelLayout = IPL_CHANNELLAYOUT_STEREO;
+    outputFormat.numSpeakers = 2;
+    outputFormat.speakerDirections = NULL;
+    outputFormat.channelOrder = IPL_CHANNELORDER_INTERLEAVED;
+
+    // Environmental Renderer and Direct Sound Effect
+    IPLerror err = iplCreateEnvironmentalRenderer(IPLContext{ nullptr,nullptr,nullptr }, environment, settings, outputFormat, NULL, NULL, envRenderer);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)